src/services/util.ts

```diff
@@ -114,6 +118,26 @@ export function sleep(time: number) {
   return new Promise(resolve => setTimeout(resolve, time));
 }
 
+export async function handleSworkTxWithLock(handler: Function) {
+  if (txLocker.swork) {
+    return {
+      status: 'failed',
```

Comment:
could be more specific, e.g. "failed: sworker locked."

Reply:
this is just for outter call judge